plugins/workspace-backpack/src/backpack.js

```diff
@@ -27,7 +26,8 @@ export class Backpack extends Blockly.DragTarget {
    * Constructor for a backpack.
    * @param {!Blockly.WorkspaceSvg} targetWorkspace The target workspace that
    *     the backpack will be added to.
-   * @param {!BackpackOptions=} backpackOptions The backpack options to use.
+   * @param {!import("./options").BackpackOptions=} backpackOptions The backpack
```

Comment:
> It says using import("./options") is the correct way to handle importing a type into JS. But we'll see what CI says about that :P

Evidently the linter disagrees.  (Maybe try single quotes?)

* AFAIK there is nothing actually checking the types here, but if our doc generation tooling uses them then you should try to figure out how to make this work.
  * Maybe try single quotes?
  * ???
  * Falling back to unused imports (with suitable suppress annotation if required) would be reasonable.
* Otherwise—if docs are not being generated, or do not suffer from inaccurate type info here—then:
  *  If we're planning to convert this to TypeScript I'd suggest adding a `// import type {BackpackOptons} from './options'` comment at the top and just use `BackpackOptions` here in the interim (that's what I did with generators in core).
  * If not, then maybe just delete the type decl entirely?